javascript/chronometer.js

```diff
@@ -1,26 +1,47 @@
 class Chronometer {
   constructor() {
+    this.currentTime = 0;
+    this.intervalId = 0;
     // ... your code goes here
   }
   startClick(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime++;
+      if (callback) {
+      callback();
+      }
+    }, 1000)// ... your code goes here
   }
   getMinutes() {
-    // ... your code goes here
+    return Math.floor((this.currentTime) / 60);// ... your code goes here
   }
   getSeconds() {
-    // ... your code goes here
+    let min = this.getMinutes();
+    let sec = this.currentTime - (min * 60);
+    return sec;
+  // ... your code goes here
   }
-  twoDigitsNumber() {
+  twoDigitsNumber(digit) {
     // ... your code goes here
+    if (digit < 10) { 
+      return "0" + digit;
+    }
+ 
+    return digit;
```

Comment:
right now it returns a number in this line, this will cause issues when invoking the function in index.js as it will try to work around a string.

Reply:
`return digit.toString()` would fix this